src-web/tableDefinitions/grcPoliciesViewDef.js

```diff
@@ -122,7 +138,7 @@ export default {
     },
   ],
```

Comment:
Now that "Disabled" is surfaced as a "Status" column, I think this `disabled` metadata can be removed.

Reply:
Ah yes. Thanks!